R/qenv-get_code.R

```diff
@@ -2,7 +2,8 @@
 #'
 #' @name get_code
 #' @param object (`qenv`)
-#' @param deparse (`logical(1)`) if the returned code should be converted to character.
+#' @param deparse (`logical(1)`) if the returned code should be converted to character
+#' @param names (`character(n)`) if provided, returns the code only for objects specified in `names`.
 #' @return named `character` with the reproducible code.
```

Comment:
Inaccurate. Return value is dependent on argument and character vector is not named.

I don't like the `deparse` argument. I would rather see smth like `as = c("character", "expression", "call")`. 
`as` > `type` >> `return`

Reply:
@chlebowa `deparse` needs to stay as it was already released on CRAN this way, we can think about depracting it. in the next run/release

Reply:
will update the `return` value, thanks

Reply:
> @chlebowa deparse needs to stay as it was already released on CRAN this way, we can think about depracting it. in the next run/release

:disappointed:

Reply:
return value is changed here f6adc39c51f2501111a9b72be6e61382f868a547
for the `deparse` let's do that after the merge https://github.com/insightsengineering/teal.code/issues/148